secrets/secrets_test.go

```diff
@@ -92,28 +90,34 @@ func TestKeeperIsClosed(t *testing.T) {
 	}
 }
 
-func TestOpenCensus(t *testing.T) {
+func TestOpenTelemetry(t *testing.T) {
+
 	ctx := context.Background()
-	te := octest.NewTestExporter(OpenCensusViews)
-	defer te.Unregister()
+
+	te := oteltest.NewTestExporter(t, OpenTelemetryViews)
 
 	k := NewKeeper(&erroringKeeper{})
 	defer k.Close()
-	k.Encrypt(ctx, nil)
-	k.Decrypt(ctx, nil)
-	diff := octest.Diff(te.Spans(), te.Counts(), "gocloud.dev/secrets", "gocloud.dev/secrets", []octest.Call{
-		{Method: "Encrypt", Code: gcerrors.Internal},
-		{Method: "Decrypt", Code: gcerrors.Internal},
+	_, _ = k.Encrypt(ctx, nil)
+
+	_, _ = k.Decrypt(ctx, nil)
+
+	// Check collected spans.
+	spanStubs := te.SpanStubs()
+	diff := oteltest.Diff(spanStubs.Snapshots(), []oteltest.Call{
```

Comment:
octest previously looks like it verified the call count and the package names as well, is it possible to do that here?

It might also be nice to call Decrypt twice, and verify it shows up twice.